packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -389,113 +389,145 @@ describe('App: Settings', () => {
   })
 
   describe('notifications', () => {
+    // Run notifications will initially be released without support for Windows
+    // https://github.com/cypress-io/cypress/issues/26786
+    const itSkipIfWindows = isWindows ? it.skip : it
+
     let setPreferencesStub
     let showSystemNotificationStub
 
-    beforeEach(() => {
-      cy.withCtx((ctx, o) => {
-        setPreferencesStub = o.sinon.stub(ctx.actions.localSettings, 'setPreferences')
-        showSystemNotificationStub = o.sinon.stub(ctx.actions.electron, 'showSystemNotification')
-        ctx.coreData.localSettings.preferences.notifyWhenRunStarts = false
-        ctx.coreData.localSettings.preferences.notifyWhenRunStartsFailing = true
-        ctx.coreData.localSettings.preferences.desktopNotificationsEnabled = true
+    context('not enabled', () => {
```

Comment:
This diff is confusing, I had to move some things around.

If you do "Hide whitespace" it is vastly more readable:

![image](https://github.com/cypress-io/cypress/assets/19196536/ba0a5e8a-fb34-4365-a767-08fb22450950)

Reply:
nice! didn't know about this option